test/13_arraycommon.jl

```diff
@@ -1797,4 +1797,26 @@ end
     end
 end
 
+@testset "repeat" begin
+    for o in (false, true), c in (false, true), i in 0:2,
+        a in [["b", "a", "b"], ["b" "a"; "b" "c"],
+              [missing, "a", "b"], ["b" "a"; missing "c"]],
+        x in [categorical(a, ordered=o, compress=c),
+              view(categorical(a, ordered=o, compress=c), axes(a)...)]
```

Comment:
Let's be even stricter just in case:
```suggestion
              view(categorical(a, ordered=o, compress=c), axes(a)...),
              view(categorical(a, ordered=o, compress=c), 1:2, 1:1)]
```